.goreleaser.yml

```diff
@@ -31,7 +37,7 @@ archives:
 
 dockers:
   - image_templates:
-      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-amd64
+      - 0xpolygon/{{ .ProjectName }}:{{ replace .Version "+" "-" }}-amd64
```

Comment:
Are we going to name CDK DAC releases with the `+cdk` suffix as well?

Reply:
for consistency, just in case, should I remove?

Reply:
Nope, good with me, I was asking more out of curiosity.